server/middleware/dbValidations.js

```diff
@@ -0,0 +1,71 @@
+import Pool from 'pg-pool';
+import validator from 'validator';
+import Helper from '../Helper/helper';
+import db from '../model/dbUsers';
+import dbloans from '../model/dbLoans';
+import dbrepayment from '../model/dbRepayment';
+import Message from '../Helper/responseMessage';
+
+const pool = new Pool({
+  user: 'victor_shagor',
+  host: 'localhost',
+  database: 'quickcredit',
+  password: 'oladimeji',
+  port: 5432,
+});
```

Comment:
Can you explain what is going on here?

Reply:
am connecting to my database locally

Reply:
I mean why you have hard-coded these credentials